EuroPythonBot/extensions/job_board/__init__.py

```diff
@@ -0,0 +1,133 @@
+"""Extension for posting in job board forum."""
+import csv
+import logging
+from pathlib import Path
+
+import aiofiles
+import attrs
+import discord
+from discord.ext import commands
+
+import configuration
+
+_logger = logging.getLogger(f"bot.{__name__}")
+
+
+async def setup(bot: commands.Bot) -> None:
+    """Set up the organisers extension."""
+    config = configuration.Config()
+    await bot.add_cog(JobBoard(bot=bot, config=config))
+
+
+@attrs.define
+class JobBoard(commands.Cog):
+    """Set up the job board extension."""
+
+    _bot: commands.Bot
+    _config: configuration.Config
+
+    @commands.Cog.listener()
+    async def on_ready(self):
```

Comment:
not fully sure I understand when it will trigger itself?

Reply:
Same! I just reused the code from the registration.
Discord magic, see https://discordpy.readthedocs.io/en/stable/api.html#discord.on_ready